docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
     volumes:
       - ./.env:/app/.env
       - ./vite.config.mjs:/app/vite.config.mjs
-      - ./src/thunderbird_accounts:/app/thunderbird_accounts
+      - ./src/thunderbird_accounts:/app/src/thunderbird_accounts
```

Comment:
Thanks! I'm not sure why i'm voluming mount at /app/thunderbird_accounts and not just /app/src/thunderbird_accounts. I'll look at this locally, but I may owe you a coffee 😄